docs/pages/kubernetes-access/guides/discovery.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Kubernetes Discovery Guides (Preview)
+description: Detailed guides for configuring Kubernetes Auto-discovery.
+layout: tocless-doc
+---
+
+## Discovery
+
+Kubernetes cluster auto-discovery is a Teleport feature that allows the Teleport process automatically discover and enroll Kubernetes clusters from different cloud providers. 
+The discovery process does not install any component on the cluster but requires Teleport to have access to the cluster API and minimal permissions.
+
+The discovery consists of two distinct steps:
+
+### Discovery
```

Comment:
```suggestion
### Polling cloud APIs
```
Or something else that's more descriptive. Since "discovery" is part of the name of the feature, I thought it would make this clearer to use a different term for the first step.